.github/workflows/create-spack-mirror.yaml

```diff
@@ -0,0 +1,33 @@
+name: create-spack-mirror
+on: workflow_dispatch
+
+jobs:
+  create-spack-mirror:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+
+      - name: create-env
+        run: |
+          source ./setup.sh
+          ./create-env.py --app all --site default --name all
```

Comment:
There is one issue here that we need to be aware of. The workflow runs for site `default`. Some sites, especially those behind firewalls, (may) have site-specific modifications of package versions. These won't be in the artifact.